ansible/roles/mesh-mgr/tasks/traefik.yml

```diff
@@ -0,0 +1,9 @@
+- name: Patch Traefik HTTP Service
+  kubernetes.core.k8s_json_patch:
+    kind: Service
+    namespace: kube-system
+    name: traefik
+    patch:
+      - op: replace
+        path: /spec/ports/0/nodePort
```

Comment:
This feels brittle, but I'm betting that the HTTP port is always the first one in the chart.